src/libPMacc/CMakeLists.txt

```diff
@@ -23,11 +23,26 @@
 ################################################################################
 # PMacc tests
 ################################################################################
-cmake_minimum_required(VERSION 3.3)
-project ("PMacc-tests")
+cmake_minimum_required(VERSION 3.1)
```

Comment:
Why you are changing this requirement? 

I think `CMAKE_CXX_STANDARD` was introduced in  cmake 3.1.3